brozzler/frontier.py

```diff
@@ -101,68 +155,35 @@ def _vet_result(self, result, **kwargs):
                         "expected %r to be %r in %r" % (k, expected, result)
                     )
 
-    def claim_sites(self, n=1):
+    def get_active_sites(self) -> List[Dict]:
+        active_sites = (
```

Comment:
is it vital to iteratie over all active sites every time?

or is there much efficiency to be gained by re-checking already claimed sites only when there are no unclaimed sites to be claimed? I guess this assumes that rethinkdb can create a claimed sites by job dict more efficently than we can in Python

Reply:
There's a level of scaling where we definitely won't want to pull down all active sites. I think for now it will be easier for us to get the logic right by pulling everything down, and we have more Brozzler Workers than we do RethinkDBs, so I'm ok with a little extra work in Python land.